### PR TITLE
Updating maven build plugin dependency versions

### PR DESCRIPTION
**Pull request by heuermh**

### CODE REVIEW

**Comment by fnothaft** [comment]
LGTM.

File: pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <parquet.version>1.8.1</parquet.version>
     <!-- Edit the following line to configure the Hadoop (HDFS) version. -->
     <hadoop.version>2.2.0</hadoop.version>
-    <scoverage.version>0.99.2</scoverage.version>
+    <scoverage.version>1.1.1</scoverage.version>
```
Comment (fnothaft) [inline pom.xml]: As an aside, have you checked the output of the `coverage` profile with this? I don't think the `coverage` profile works now, would be interesting to see if this fixed it. If not, we should open an issue to tackle later.

Comment (heuermh) [inline pom.xml]: Will check it out shortly

Comment (heuermh) [inline pom.xml]: No it does not, let me try with ```0.99.10``` . . .

Comment (heuermh) [inline pom.xml]: I see as of ```0.99.2``` support for scala 2.10 was dropped.
https://github.com/scoverage/scalac-scoverage-plugin/blob/master/2.10.md

Updating to ```1.1.1``` as this pull request does results in a maven transitive dependency error.

Versions ```0.99.2``` and ```0.99.10``` fail with
```adam-core/target/scoverage.coverage.xml (No such file or directory)```

Downgrading to ```0.98.2``` fails with
```adam-core/target/scala-2.10.4/classes/scoverage.coverage (No such file or directory)```

Comment (fnothaft) [inline pom.xml]: I've opened #849 for this.

**Comment by ryan-williams** [comment]
this lgtm, thanks @heuermh. I love the feeling of all of these tools improving under us :)

**Comment by fnothaft** [comment]
Thanks @heuermh! Rebased and merged as https://github.com/bigdatagenomics/adam/commit/bbb0c32266250057a95e2cdab647d38832729609.

**Pull request closed without merging**